doc/api/child_process.markdown

```diff
@@ -128,9 +127,9 @@ be sent `'SIGTERM'`. See `signal(7)` for a list of available signals.
 
 May emit an `'error'` event when the signal cannot be delivered. Sending a
 signal to a child process that has already exited is not an error but may
-have unforeseen consequences: if the PID (the process ID) has been reassigned
-to another process, the signal will be delivered to that process instead.
-What happens next is anyone's guess.
+have unforeseen consequences. Specifically, if the process ID (PID) has been
```

Comment:
Maybe write out `process identifier`? I'm not a fan of `ID` as it isn't even an abbreviation.